tests/testthat/test-bookmarking.R

```diff
@@ -32,16 +30,16 @@ test_that("Inputs and values in query string", {
   expect_identical(as.list(vals$values), list())
 
   # Multiple instances of _inputs_ or _values_
-  suppress_stacktrace(expect_warning(RestoreContext$new("?_inputs_&a=1&_inputs_")))
-  suppress_stacktrace(expect_warning(RestoreContext$new("?_inputs_&a=1&_inputs_&")))
-  suppress_stacktrace(expect_warning(RestoreContext$new("?_inputs_&a=1&_inputs_&b=2")))
-  suppress_stacktrace(expect_warning(RestoreContext$new("?_inputs_&a=1&_values_&b=2&_inputs_&")))
-  suppress_stacktrace(expect_warning(RestoreContext$new("?_values_&a=1&_values_")))
-  suppress_stacktrace(expect_warning(RestoreContext$new("?_inputs_&a=1&_values_&_values&b=2")))
+  suppress_stacktrace(expect_warning(expect_warning(RestoreContext$new("?_inputs_&a=1&_inputs_"))))
```

Comment:
Why is there `expect_warning(expect_warning(...))`?

Reply:
Previously `expect_warning()` swallowed all warnings (well, technically all conditions), and these calls made two warnings (I looked into why a little bit but couldn't see an obvious reason).